assignment02/index.html

```diff
@@ -1,53 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-    <head>
-        <title>Assignment 2</title>
-		<meta charset="utf-8">
-       
-        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-        <script src="script.js" ></script>
-        
-
-    </head>
-    <body>
-	<div id= wrapper>
-        <!--     
-            Deepa Natarajan
-            assignment01.htm
-            INFO_2340_WW
-            Ms.Thoendel
-            12/14/2019
-        -->
-		<header>
-        <h1>The Wizard of OZ! </h1>
-		</header>
-		
-		<main>
-		<div id=format>
-		<p>
-        <strong><span id=lady></span></strong> lived in the midst of the great <strong><span id=state></span></strong> ,
```

Comment:
best practice is to place the value of the id in quotation marks (eithier single or double) - i.e. id="lady"